daemon/cmd/daemon_main.go

```diff
@@ -250,9 +250,8 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.MarkDeprecated(option.EnableRuntimeDeviceDetection, "Runtime device detection and datapath reconfiguration is now the default and only mode of operation")
 
 	flags.String(option.DatapathMode, defaults.DatapathMode,
-		fmt.Sprintf("Datapath mode name (%s, %s, %s, %s)",
-			datapathOption.DatapathModeVeth, datapathOption.DatapathModeNetkit,
-			datapathOption.DatapathModeNetkitL2, datapathOption.DatapathModeLBOnly))
+		fmt.Sprintf("Datapath mode name (%s, %s, %s)",
+			datapathOption.DatapathModeVeth, datapathOption.DatapathModeNetkit, datapathOption.DatapathModeNetkitL2))
```

Comment:
btw: do we need to cleanup the helm values too? There are at least occurrences in the doc.

https://github.com/cilium/cilium/blob/pr/lb-improvements/install/kubernetes/cilium/values.yaml#L627-L627

And maybe it's also worth to use the new flag `--bpf-lb-only` when `loadbalancer.standalone=true` in the cilium-configmap (https://github.com/cilium/cilium/blob/pr/lb-improvements/install/kubernetes/cilium/templates/cilium-configmap.yaml#L785-L785)?